packages/TorneloScoresheet/src/components/OptionPopup/OptionPopup.tsx

```diff
@@ -0,0 +1,66 @@
+import { TouchableOpacity, View } from 'react-native';
+import { SvgProps } from 'react-native-svg';
+import RoundedView from '../RoundedView/RoundedView';
+import Sheet from '../Sheet/Sheet';
+import { styles } from './style';
+import { colours } from '../../style/colour';
+import PrimaryText, { FontWeight } from '../PrimaryText/PrimaryText';
+import React from 'react';
+
+type Option = {
```

Comment:
Instead of option would something like popupContent be better? If you prefer option I'm happy to approve it :)

Reply:
I'd say option makes sense, since we're not defining the JSX content of option buttons here.